core/deployment/src/main/java/io/quarkus/deployment/builditem/ExcludeDependencyBuildItem.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Build item that defines dependencies that should be indexed. This can be used when a dependency does not contain
```

Comment:
Hm, quite the contrary... or?

Reply:
Yes, I should avoid pushing after 5PM....

Reply:
I can still see the old version of the javadoc...